assets/src/block-editor/components/with-media-library-notice.js

```diff
@@ -57,47 +56,82 @@ export default ( InitialMediaUpload, minImageDimensions ) => {
 		 */
 		initFeaturedImage() {
 			const FeaturedImageSelectMediaFrame = getSelectMediaFrame( FeaturedImageToolbarSelect );
+
+			const FeaturedImageLibrary = wp.media.controller.FeaturedImage.extend( {
+				defaults: {
+					...wp.media.controller.FeaturedImage.prototype.defaults,
+					date: false,
+					filterable: false,
+					// Note: These suggestions are shown in the media library image browser.
+					suggestedWidth: EXPECTED_WIDTH,
+					suggestedHeight: EXPECTED_HEIGHT,
+				},
+			} );
+
 			this.frame = new FeaturedImageSelectMediaFrame( {
 				allowedTypes: this.props.allowedTypes,
-				button: {
-					text: __( 'Select', 'amp' ),
-					close: false,
-				},
-				states: [
-					new wp.media.controller.Library( {
-						title: __( 'Choose image', 'amp' ),
-						library: wp.media.query( { type: 'image' } ),
-						multiple: false,
-						date: false,
-						priority: 20,
-						// Note: These suggestions are shown in the media library image browser.
-						suggestedWidth: EXPECTED_WIDTH,
-						suggestedHeight: EXPECTED_HEIGHT,
-					} ),
-				],
+				state: 'featured-image',
+				states: [ new FeaturedImageLibrary(), new wp.media.controller.EditImage() ],
```

Comment:
I've added the `EditImage` controller here to the state since it was a missing feature.